app/templates/pages/justice-and-other-user.html

```diff
@@ -13,26 +13,28 @@
 {% endblock %}
 
 {% block content %}
-<h1 class="govuk-heading-l">GitHub Organisation Invitation</h1>
+<h1 class="govuk-heading-l">Verify your work email address</h1>
 
 <p class="govuk-body">
-  To access the Ministry of Justice GitHub organisations you've selected, your email address <strong>{{email}}</strong>
-  will be verified
-  using Azure Active Directory (AD). This secure step is essential to confirm your identity and initiate the process of
-  joining our GitHub organisations.
+  This secure step is essential to confirm your identity and initiate the process of joining our GitHub organisations. <strong>{{email}}</strong>
```

Comment:
This paragraph still contains the old text. That needs to be removed. You need to remove everything from 

> <strong>{{email}}</strong> will be verified using Azure Active Directory (AD).
>   This secure step is essential to confirm your identity and initiate the process of joining our GitHub organisations.

The paragraph should just read:

> This secure step is essential to confirm your identity and initiate the process of joining our GitHub organisations.

Reply:
yup i just removed the duplicate line.

Reply:
After the first line it still says "<strong>{{email}}</strong>". Please remove.